Tasks/AzureVmssDeployment/blobservice.ts

```diff
@@ -1,3 +1,5 @@
+import * as artifactProviders from 'artifact-mover/Providers';
```

Comment:
I was going through the file. Few comments:
1. _getArchiveFileExtension is unused.
2.  does compress have 7z dependency? Consider using what  WebApp task is using instead